CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## DLA-Future 0.7.2
+
+### Changes
+
+- Changed C ScaLAPACK API indexing convention to 1-based for partial eigenspectrum. (#1248)
```

Comment:
Is this a bugfix? I'm fine either way and won't matter too much especially if we mark 0.7.0 and 0.7.1 deprecated in spack, but by making this release 0.7.2 and not 0.8.0 we're signaling that this is a bugfix.

Reply:
I wondered the same, but I went with "changed" because it was not technically a bug. Happy to mark it as a bugfix if preferred.

Reply:
It's not a strong preference from me. Mostly wondering out loud...